### ISSUE TITLE
chown: invalid user: 'apache:apache'

### ISSUE THREAD

**Issue by rngadam** [GluuFederation/community-edition-setup#56]
Ubuntu 14.04 from packages, from the setup.log

```
09:44:30 09/04/15 Running: /bin/chown apache:apache /etc/certs/httpd.key.orig
09:44:30 09/04/15 /bin/chown: invalid user: 'apache:apache'

09:44:30 09/04/15 Running: /bin/chmod 700 /etc/certs/httpd.key.orig
09:44:30 09/04/15 Running: /bin/chown apache:apache /etc/certs/httpd.key
09:44:30 09/04/15 /bin/chown: invalid user: 'apache:apache'
```

**Comment by mzico** [comment]
Ingredients of /etc/certs/* should be tomcat:tomcat

**Comment by yurem** [comment]
Fixed in CE 3.0.0 Alpha
